MeshFeatureTracking/src/MeshFeatureTrackingComponent.cpp

```diff
@@ -242,7 +242,7 @@ void MeshFeatureTrackingComponent::setData( const Ra::Engine::Renderer::PickingR
         return;
     }
     // if lines, retrieve triangle-based indices
-    const auto& t = m_pickedMesh->getCoreGeometry().m_indices;
+    const auto& t = m_pickedMesh->getCoreGeometry().getIndices();
```

Comment:
this should be getIndicesWithLock()
I'll do it in a PR to update engine layout, maybe for this PR you could stick to CI ?
Or do we want to have a version valid for pre engine layout modification ?

Reply:
oops this one is ok with get Indices ;)